Assets/Scripts/Flow/NoonIsland.cs

```diff
@@ -1,31 +1,59 @@
 using Cinemachine;
-using UnityEngine;
 using Popups;
+using UnityEngine;
 using UnityEngine.SceneManagement;
 
 public class NoonIsland : MonoBehaviour
 {
-    [Header("UI")] 
-    [SerializeField] private UI ui;
-    
-    [Header("Cameras")] 
-    [SerializeField] private Camera mainCamera;
-    [SerializeField] private CinemachineVirtualCamera virtualCamera;
-    
-    [Header("Player")] 
-    [SerializeField] private PlayerController playerController;
-    [SerializeField] private Vector3 spawnPosition;
-    
-    [Header("Managers")] 
-    [SerializeField] private BarrierManager barrierManager;
-    [SerializeField] private AudioManager audioManager;
-    [SerializeField] private InventoryManager inventoryManager;
-    [SerializeField] private EnergyManager energyManager;
-    [SerializeField] private QuestManager questManager;
-    [SerializeField] private DialogueManager dialogueManager;
-    [SerializeField] private SpriteManager spriteManager;
-    [SerializeField] private DayManager dayManager;
-    [SerializeField] private WaterManager waterManager;
+    [Header("UI")]
+    [SerializeField]
+    private UI ui;
+
+    [Header("Cameras")]
+    [SerializeField]
+    private Camera mainCamera;
+
+    [SerializeField]
+    private CinemachineVirtualCamera virtualCamera;
+
+    [Header("Player")]
+    [SerializeField]
+    private PlayerController playerController;
+
+    [SerializeField]
+    private Vector3 spawnPosition;
+
+    [Header("Managers")]
+    [SerializeField]
+    private BarrierManager barrierManager;
+
+    [SerializeField]
+    private AudioManager audioManager;
+
+    [SerializeField]
+    private InventoryManager inventoryManager;
+
+    [SerializeField]
+    private EnergyManager energyManager;
+
+    [SerializeField]
+    private QuestManager questManager;
+
+    [SerializeField]
+    private DialogueManager dialogueManager;
+
+    [SerializeField]
+    private SpriteManager spriteManager;
+
+    [SerializeField]
+    private DayManager dayManager;
+
+    [SerializeField]
+    private WaterManager waterManager;
+
+    [Header("Level")]
+    [SerializeField]
+    private Level Noonisland_level;
```

Comment:
Apart from fixing the snake casing, what is this doing actually? I can't really see the prefab unless I checkout the branch
```suggestion
    private Level NoonIslandLevel;
```